ansible/adhoc/migrate-opendistro.yml

```diff
@@ -0,0 +1,18 @@
+# Migrates data from containerised opendistro v1.12.0 to containerised opensearch 2.1.0
+#
+# Usage:
+#
+#   git pull                # and merge etc as required, including changing groups file to define opensearch not opendistro
+#   ansible-playbook ansible/adhoc/migrate-opendistro.yml
+#   ansible-playbook ansible/monitoring.yml
+#
+# This does not automatically remove the opendistro podman volume. That can be done manually once happy data has transferred ok.
+
+- hosts: opensearch
+  become: no
+  gather_facts: no
+  tasks:
+    - name: Export opendistro volume
+      import_role:
+        name: opensearch
+        tasks_from: migrate-opendistro.yml
```

Comment:
Can this be run as part of the role/tasks/main.yml rather than an adhoc playbook? Existing Azimuth-deployed clusters would need this ad-hoc playbook to run as part of an upgrade.